Deep_learning/Deep-learning.rst

```diff
@@ -372,18 +372,18 @@ This step is optional but most PyTorch scripts will use torchvision to load mode
 Option 3: Install using minimal ROCm docker file
 ************************************************
 
-1. Download pytorch dockerfile:
+1. Download dockerfile based on the OS choose:
+Recommend to use - Dockerfile-<OS distro>-complete to get all the ROCm Math libs installed which are required for PyTorch.
 
-`Dockerfile <https://github.com/ROCmSoftwarePlatform/pytorch/wiki/Dockerfile>`_
+`Dockerfile <https://github.com/RadeonOpenCompute/ROCm-docker/tree/master/dev>`_
```

Comment:
The first dockerfile also has a PyTorch build step at the end. The latter one is simply to install ROCm afaict.